### ISSUE TITLE
Missing code signature in Kap version 3.4.1

### ISSUE THREAD

**Issue by suschizu** [wulkano/Kap#1046]
I've received a warning that the macOS code signature in version 3.4.1 is missing (expected: Wulkano AS)- can you confirm that?
Thanks!

**Comment by skllcrn** [comment]
On it!

**Comment by eleijonmarck** [comment]
yea, it goes to the bin right away :(

**Comment by sindresorhus** [comment]
https://github.com/wulkano/Kap/releases/tag/v3.4.2

**Comment by core-code** [comment]
thanks for the effort. doesn't look good however. you've used the "Apple Distribution" certificate which is only meant to be used for uploading to the Mac App Store. for distributing directly to users the "Developer ID Application" certificate needs to be used. and incidentally this one was used up to version 3.4.0...

**Comment by skllcrn** [comment]
I am sorry about this. Meanwhile, I hope this workaround is helpful: https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac

**Comment by prewk** [comment]
I just downloaded Kap `v3.4.2` from the website and that's not properly signed either?

**Comment by mesqueeb** [comment]
In case anyone wants to download version 3.4.0, you can do so from the releases here:
https://github.com/wulkano/Kap/releases/tag/v3.4.0

**Comment by prewk** [comment]
> In case anyone wants to download version 3.4.0, you can do so from the releases here: https://github.com/wulkano/Kap/releases/tag/v3.4.0

Getting the same alert for this one haha. Are there problems at my end?

**Comment by aulneau** [comment]
> I am sorry about this. Meanwhile, I hope this workaround is helpful: https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac

I don't think this is really helpful -- the point of signed apps is not to simply allow unsigned apps to be installed.

> In case anyone wants to download version 3.4.0, you can do so from the releases here: https://github.com/wulkano/Kap/releases/tag/v3.4.0

This unfortunately does not work on the latest version of macos.

@sindresorhus @skllcrn this has been an ongoing issue for more than a month. Many people are bummed out by this -- is the app no longer being actively developed?

**Comment by skllcrn** [comment]
I'm in touch with Apple support, I do not have a timeline. Getting either errors or nothing at all when attempting to generate the certificate we need... While I too understand the point signing the releases, hopefully the workaround is helpful for now.

**Comment by erawhctim** [comment]
If anyone is looking for a replacement to use in the meantime (albeit, a less feature-rich option), [macOS supports screen recordings natively](https://support.apple.com/en-us/HT208721).

Thanks for your hard work @skllcrn 🙂

**Comment by skllcrn** [comment]
Let me know if you need help opening the app @erawhctim!

**Comment by eleijonmarck** [comment]
> If anyone is looking for a replacement to use in the meantime (albeit, a less feature-rich option), [macOS supports screen recordings natively](https://support.apple.com/en-us/HT208721).
> 
> Thanks for your hard work @skllcrn 🙂

I never knew this, this is amazing. The only thing is that they can have a gif from it.

It also records mic and mouse clicks. works for me directly actually

**Comment by mesqueeb** [comment]
@eleijonmarck don't do it. You'll end up with screenrecording files 100s of times bigger than KAP. The Filesize is the biggest downfall. 

KAP's Filesize is tiny, perfect for sharing.

**Comment by keithly** [comment]
This is still a problem. I can no longer use any recent version of kap on macOS 11.6.1 Intel (I haven't tried every version.) 3.4.1 and older have the code signing error, and later versions have the malicious software check error.

**Comment by jegli** [comment]
I had the same "issue" (Kap v3.5.1, Apple M1, macOS 12.1 Monterey) but if you just click on the small question-mark and follow the instructions you might get it to work. It worked for me. Here the instructions:
```
In the Finder on your Mac, locate the app you want to open.

1. Don’t use Launchpad to locate the app.

2. Control-click the app icon, then choose Open from the shortcut menu.

3. Click Open.

The app is saved as an exception to your security settings, and you can open it at any time by double-clicking it, just as you do with any registered app.
```

**Comment by jplomas** [comment]
May or may not help, but I used SD Notary to help automate the code signing of an Electron-based app for Apple M1/Monterey which also kept failing until I went into the Advanced tab and ticked all the additional permissions... then all worked like magic again 🎉

SD Notary 1.4.8 Build 61: https://s3.amazonaws.com/latenightsw.com/SDNotary1.4.8-61.zip
Blog and instructions: https://latenightsw.com/sd-notary-notarizing-made-easy/
